__tests__/steps/navSteps.spec.ts

```diff
@@ -66,6 +67,16 @@ describe('navSteps', () => {
       expect(getActiveStepsFromNavigationSteps()).toEqual({});
     });
 
+    it('it should run the navigationbased active steps', () => { 
+      config.onMarkStep = () => {};
+      spy = jest.spyOn(config, 'onMarkStep');
```

Comment:
`navigationbased`? you are testing that `onMarkStep` is called, not sure what `navigationbased` means here.

Reply:
a bit more specific description here maybe like `should call the config's 'onMarkStep' when the function 'markStep' is being called`